README.md

```diff
@@ -39,3 +38,6 @@ A collection of code examples which generate modelling features for a mobile pho
 | [sum_of_open_and_late_loan_balances](lib/sum_of_open_and_late_loan_balances) | proxy for outstanding debt. sum of open and late-payment loan balance                                                                  |
 | [sum_of_default_loan_balances](lib/sum_of_default_loan_balances)             | sum of default loan balances                                                                                                           |
 | [debt_to_income_ratio_latest](lib/debt_to_income_ratio_latest)               | ratio between the sum of the non-closed tradline balances and sum of credit transactions across all depository accounts over 0-30 days |
+| [avg_daily_balance_0_30](lib/average_daily_balance)                               | average daily balance. average daily balance across all depository accounts over 0-30 days  |
```

Comment:
nit: can we be consistent with using `avg` or `average` across the feature and function names? Soft preference for `average` but I think consistency is more important.

Reply:
Looks like we're still mixing conventions after https://github.com/pngme/pngme-feature-library/pull/16/commits/06a1c98cad1c024a3497791aad378064c0f217fa.

Can we replace all instances of `avg` with `average` (or vice versa) so naming is consistent? e.g. `get_avg_end_of_day_balance`